README.md

```diff
@@ -200,6 +200,10 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
        - Static Sites with [GitLab Pages](https://about.gitlab.com/product/pages).
        - Container Registry with a 10 GB limit per repo.
        - Project Management and issue Tracking.
+  [[framagit.com](https://framagit.org/)](https://framagit.org/) — Framagit est la forge logicielle de Framasoft reposant sur le logiciel Gitlab. :
```

Comment:
Please use english only for descriptions, I think this is small enough list of features that you can just make it one line.